rewrite-maven/src/main/java/org/openrewrite/maven/internal/MavenPomDownloader.java

```diff
@@ -516,6 +516,14 @@ public Pom download(GroupArtifactVersion gav,
                             RawPom rawPom = RawPom.parse(fis, Objects.equals(versionMaybeDatedSnapshot, gav.getVersion()) ? null : versionMaybeDatedSnapshot);
                             Pom pom = rawPom.toPom(inputPath, repo).withGav(resolvedGav);
 
+                            if("jar".equals(pom.getPackaging())){
```

Comment:
```suggestion
                            if (pom.getPackaging() == null || "jar".equals(pom.getPackaging())) {
```

Reply:
Note that `jar` is the default packaging. So we should also check that the jar exists, when the value is `null`.